src/solutions/appointments/request.rs

```diff
@@ -0,0 +1,53 @@
+// GENERATED CODE
+
+use crate::api_default_imports::*;
+
+resource_api_client!(
+    AppointmentsApiClient,
+    AppointmentsIdApiClient,
+    ResourceIdentity::Appointments
+);
+
+impl AppointmentsApiClient {
+    post!(
+        doc: "Create new navigation property to appointments for solutions",
+        name: create_appointments,
+        path: "/appointments",
+        body: true
+    );
+    get!(
+        doc: "Get appointments from solutions",
+        name: list_appointments,
+        path: "/appointments"
+    );
+    get!(
+        doc: "Get the number of the resource",
+        name: appointments,
```

Comment:
Update this to `get_appointments_count` by updating codgen here: https://github.com/sreeise/graph-rs-sdk/blob/b35f98e641ed17ad13f21c5729d1ffd8894394cc/graph-codegen/src/settings/method_macro_modifier.rs#L55